generation/WinSDK/enums.json

```diff
@@ -30974,5 +30974,524 @@
+      }
+    ],
+    "uses": [
+      {
+        "struct": "IDLDESC",
+        "field": "wIDLFlags"
+      }
+    ]
+  },
+  {
+    "name": "NUMPRS_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "NUMPRS_",
+      "header": "oleauto.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "NUMPARSE",
+        "field": "dwInFlags"
+      },
+      {
+        "struct": "NUMPARSE",
+        "field": "dwOutFlags"
+      }
+    ]
+  },
+  {
+    "name": "PICTYPE",
+    "type": "int",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "PICTYPE_",
+      "header": "olectl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "PICTDESC",
+        "field": "picType"
+      }
+    ]
+  },
+  {
+    "name": "VARCMP",
+    "autoPopulate": {
+      "filter": "VARCMP_",
+      "header": "oleauto.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "VarCmp",
+        "parameter": "return"
+      },
+      {
+        "method": "VarCyCmp",
+        "parameter": "return"
+      },
+      {
+        "method": "VarCyCmpR8",
+        "parameter": "return"
+      },
+      {
+        "method": "VarDecCmp",
+        "parameter": "return"
+      },
+      {
+        "method": "VarDecCmpR8",
+        "parameter": "return"
+      },
+      {
+        "method": "VarR4CmpR8",
+        "parameter": "return"
+      }
+    ]
+  },
+  {
+    "name": "PASTE_SPECIAL_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "PSF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "OLEUIPASTESPECIALA",
+        "field": "dwFlags"
+      },
+      {
+        "struct": "OLEUIPASTESPECIALW",
+        "field": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "EMBDHLP_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "EMBDHLP_",
+      "header": "ole2.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OleCreateEmbeddingHelper",
+        "parameter": "flags"
+      }
+    ]
+  },
+  {
+    "name": "FDEX_PROP_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "fdexProp",
+      "header": "dispex.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "interface": "IDispatchEx",
+        "method": "GetMemberProperties",
+        "parameter": "pgrfdex"
+      }
+    ]
+  },
+  {
+    "name": "LOAD_PICTURE_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "LP_",
+      "header": "olectl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OleLoadPictureFileEx",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OleLoadPictureEx",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "type": "uint",
+    "addUsesTo": "CTRLINFO",
+    "members": [],
+    "uses": [
+      {
+        "struct": "CONTROLINFO",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "OLE_CREATE",
```

Comment:
OLE_CREATE_FLAGS

Reply:
EIther that, or OLECREATE to be consistent with OLERENDER which is an existing enum in the headers.